src/services/mockFields.ts

```diff
@@ -163,6 +163,54 @@ const FIELDS_CONFIG: Fields = [
     groups: ['relationships'],
     scope: ['/patient/:uid'],
     accessLevel: []
+  },
+  {
+    fieldName: 'process',
+    fieldType: 'String',
+    fieldLabel: 'Process',
+    groups: ['audit_trail'],
+    scope: ['/patient/:uid', '/patient/:uid/audit-trail'],
+    accessLevel: []
+  },
+  {
+    fieldName: 'actionTaken',
+    fieldType: 'String',
+    fieldLabel: 'Action taken',
+    groups: ['audit_trail'],
+    scope: ['/patient/:uid', '/patient/:uid/audit-trail'],
+    accessLevel: []
+  },
+  {
+    fieldName: 'links',
+    fieldType: 'String',
+    fieldLabel: 'Links',
+    groups: ['audit_trail'],
+    scope: ['/patient/:uid', '/patient/:uid/audit-trail'],
+    accessLevel: []
+  },
+  {
+    fieldName: 'when',
+    fieldType: 'Date',
+    fieldLabel: 'When',
+    groups: ['audit_trail'],
+    scope: ['/patient/:uid', '/patient/:uid/audit-trail'],
+    accessLevel: []
+  },
+  {
+    fieldName: 'changedBy',
+    fieldType: 'String',
+    fieldLabel: 'Changed by',
+    groups: ['audit_trail'],
+    scope: ['/patient/:uid', '/patient/:uid/audit-trail'],
+    accessLevel: []
+  },
+  {
+    fieldName: 'comment',
+    fieldType: 'String',
+    fieldLabel: 'Comment',
+    groups: ['audit_trail'],
+    scope: ['/patient/:uid', '/patient/:uid/audit-trail'],
+    accessLevel: []
   }
```

Comment:
We don't have to specify the audit trail fields here. This is only applicable for the patient record fields. This is meaning, no matter what the country is (the implementation config), the audit trail columns will still be the same.